code/lightna/lightna-engine/App/Storage/Opcache.php

```diff
@@ -21,7 +21,7 @@ protected function init(array $options): void
         $this->dir = $this->dir . '/';
     }
 
-    public function set(string $key, mixed $value): void
+    public function set(string $key, mixed $value, array $tags = []): void
```

Comment:
ha have question left, how it behaves if clean million items, will it fail or stuck?